### PR TITLE
Reimplement webkiosk auth

### PR DESCRIPTION
**Pull request by ryzokuken**

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -49,26 +50,27 @@ func SuccessResponse() map[string]string {
...
 		log.Fatal(err)
 	}
 
-	// TODO: @dwaipayan FIXME: Make cases for successful login, invalid dob
-	//                         invalid password etc.
 	respBody, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Println(string(respBody[:]))
```
Comment (yashmehrotra) [inline utils/utils.go]: Remove this

File: utils/utils.go [hunk 2]
```diff
@@ -49,26 +50,27 @@ func SuccessResponse() map[string]string {
...
-	return true, "error"
+
+	if strings.Contains(string(respBody), "Invalid Password") {
+		return false, "Invalid Password"
```
Comment (yashmehrotra) [inline utils/utils.go:75]: What happens if enrollment no is wrong ?

File: api/user_api.go [hunk 3]
```diff
@@ -53,9 +52,9 @@ func WebkioskAuth(c echo.Context) error {
 
 	enroll := payload["enrollment_number"]
 	password := payload["password"]
-	dob := payload["dob"]
+	// dob := payload["dob"]
```
Comment (yashmehrotra) [inline api/user_api.go]: Lets remove `DOB`

**Review by yashmehrotra — CHANGES_REQUESTED** [review]

File: utils/utils.go (hunk 2 above)
Comment (ryzokuken) [reply utils/utils.go:75]: like...if it is invalid? I guess it's best checked on the frontend. Or maybe look into this yourself, maybe it doesn't throw a session timeout for you.

Comment (yashmehrotra) [reply utils/utils.go:75]: Can you do a check for success response rather than checking for unsuccessful, that would be better

**Pull request merged**